core/storage/exploration/gae_models.py

```diff
@@ -78,6 +78,9 @@ class ExplorationModel(base_models.VersionedModel):
     # The list of parameter changes to be performed once at the start of a
     # reader's encounter with an exploration.
     param_changes = ndb.JsonProperty(repeated=True, indexed=False)
+    # A boolean indicating whether automatic text-to-speech is enabled in
+    # this exploration.
+    auto_tts_enabled = ndb.BooleanProperty(default=True, indexed=False)
```

Comment:
indexed=True (might as well. Easier to introspect later on if needed)

Reply:
done